app/functions.R

```diff
@@ -120,44 +119,46 @@ renderScoreHtml <- function(risk) {
-    } else {
-      # after efficacy threshold
-      text <- tags$p("Congratulations on receiving", 
-                     ifelse(vaccines[[vaccine]]$doses==1, "the ", "both doses of the "), 
-                     vaccine_labels[vaccine], " COVID-19 vaccine!",
-                     "This vaccine reduces your risk of contracting symptomatic COVID-19 by ",
-                     formatPercent(vaccines[[vaccine]]$efficacy), ". ",
-                     tags$a("Click here ", href=urls$cdc_vaccines), 
-                     "for more information about the United States' vaccination program.")
+  } else if (!is.na(months_last_vaccination)) {
+    # if has vaccination
+    infection_efficacy_perc <- formatPercent(risk$vaccine_reduction$infection_efficacy)
+    hosp_efficacy_perc <- formatPercent(risk$vaccine_reduction$hosp_efficacy)
+    icu_efficacy_perc <- formatPercent(risk$vaccine_reduction$icu_efficacy)
+    death_efficacy_perc <- formatPercent(risk$vaccine_reduction$death_efficacy)
+    
+    if (months_last_vaccination == "lt_1mo"){
+      # last vaccine within 1 month 
+      months_last_vaccination_text <- HTML(paste0("Congratulations on receiving the ", vaccine_labels[vaccine], " COVID-19 vaccine! ", 
+                     "The vaccine reaches its full efficacy at around 14 days after vaccination. ",
+                     "We estimate that the full efficacy of the ", vaccine_labels[vaccine]))
+    } else if (months_last_vaccination == "btw_1_6mo"){
+      # last vaccine within 1-6 month
+      months_last_vaccination_text <- HTML(paste0("Congratulations on receiving the ", vaccine_labels[vaccine], " COVID-19 vaccine! ",
+                                             "Between 1 and 6 months after last vaccination, we estimate that the efficacy of the ", vaccine_labels[vaccine]))
+      
+    } else if (months_last_vaccination == "gt_6mo"){
+      # last vaccine within greater than 6 month
+      months_last_vaccination_text <- HTML(paste0("Congratulations on receiving the ", vaccine_labels[vaccine], " COVID-19 vaccine! ",
+                     "Be sure check if you are eligible to receive a booster shot. ",
+                     "More than 6 months after last vaccination, we estimate that the efficacy of the ", vaccine_labels[vaccine]))
     }
+    
+    reduction_text <- HTML(paste0( " COVID-19 vaccine is a ",  
+                             infection_efficacy_perc, "reduction in risk of infection; a ", 
+                             hosp_efficacy_perc, "reduction in risk of hospitalization; a ", 
+                             infection_efficacy_perc, "reduction in risk of ICU addmission; and a ", 
+                             infection_efficacy_perc, "reduction in risk of death due to COVID-19. ", 
+                             tags$a("Click here ", href=urls$cdc_vaccines), "for more information about the United States' vaccination program."))
+    
+    text <- tags$p(months_last_vaccination_text, reduction_text)
```

Comment:
Can you remove line 168 and onward? "Early evidence from vaccine trials suggests that some or all of the vaccines may provide additional protection against hospitalization and death, however this conclusion is only preliminary due to the small number of individuals in the treatment groups who contracted COVID-19." This was leftover from a long time ago.